.github/workflows/main_pull.yml

```
@@ -0,0 +1,106 @@
+name: CI
+
+permissions: read-all
+
+on:
+  pull_request:
+    branches:
+      - develop
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - labeled
+
+  # Allows to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name || github.run_id }}
```

The expression can be simplified. As this is running on  pull requests only, `github.head_ref` is always set and there is no need for fallbacks:
```suggestion
  group: ${{ github.workflow }}-${{ github.head_ref }}
```

---

https://github.com/Activiti/Activiti/pull/4529/commits/d5f96261875a6604378c59939a4d890b91061612